docs/reference/components/sinks/redis.cue

```diff
@@ -0,0 +1,127 @@
+package metadata
+
+components: sinks: redis: {
+	title: "Redis"
+	classes: {
+		commonly_used: false
+		delivery:      "best_effort"
+		development:   "beta"
+		egress_method: "stream"
+		service_providers: []
+		stateful: false
+	}
+	features: {
+		buffer: enabled:      false
+		healthcheck: enabled: true
+		send: {
+			compression: enabled: false
+			encoding: {
+				enabled: true
+				codec: {
+					enabled: true
+					default: "json"
```

Comment:
```suggestion
```

More recently we've started to leave off the defaults for encoding as users were confused about why their events were encoded in a given way. Requiring users to choose should make them more aware and allow them to choose the correct encoding for their use-case.